ratelimit/keeper/hooks.go

```diff
@@ -4,18 +4,14 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
-	epochstypes "github.com/Stride-Labs/stride/v17/x/epochs/types"
 )
 
 // Before each hour epoch, check if any of the rate limits have expired,
 // and reset them if they have
-func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
-	if epochInfo.Identifier == epochstypes.HOUR_EPOCH {
-		epochHour := uint64(epochInfo.CurrentEpoch)
-
+func (k Keeper) BeginBlocker(ctx sdk.Context) {
```

Comment:
The `BeginBlocker` should not be in `hooks.go` as it's no longer a hook. Either add these lines directly to `BeginBlock` in `module.go` or move the `BeginBlocker` to an `abci.go` file (see https://github.com/cosmos/cosmos-sdk/blob/v0.47.8/x/distribution/abci.go).

Reply:
you beat me to it haha, was planning to do this in a separate PR to make the diffs easier to review. 

Just pushed it to this PR